ckcp/openshift_dev_setup.sh

```diff
@@ -327,7 +327,7 @@ patches:
 
   echo "- Setup kcp access:"
   "$PROJECT_DIR/images/access-setup/content/bin/setup_kcp.sh" \
-    "$DEBUG" \
+    ${DEBUG:+"$DEBUG"} \
```

Comment:
I can do that, but just so that you're aware it won't change the need for `${DEBUG:+"$DEBUG"}` because if you just use `"$DEBUG"` and DEBUG is declared but empty. The scripts fails since we're sending a null parementer.

Reply:
You're right.